local-volume/provisioner/pkg/util/volume_util.go

```diff
@@ -106,14 +106,17 @@ func (u *volumeUtil) DeleteContents(fullPath string) error {
 
 // GetFsAvailableByte returns available capacity in byte about a mounted filesystem.
 // fullPath is the pathname of any file within the mounted filesystem. Capacity
-// returned here is total capacity available to non-root users, and does not include
-// fs reserved space.
+// returned here is total capacity available.
 func (u *volumeUtil) GetFsAvailableByte(fullPath string) (uint64, error) {
-	var s unix.Statfs_t
-	if err := unix.Statfs(fullPath, &s); err != nil {
+	//largely copied from kubernetes/pkg/volume/util/fs.go:FsInfo(path)
```

Comment:
I had some vendoring issues related to log_dir being defined twice, by glog being imported twice.

For example, 
`panic: /var/folders/59/yc7_f4fd53nbyw868zqpk78cn7shx9/T/go-build393895106/github.com/kubernetes-incubator/external-storage/local-volume/provisioner/pkg/discovery/_test/discovery.test flag redefined: log_dir
`

I was able to resolve it by locally executing
`glide install --strip-vendor --strip-vcs`

by following the instructions [here on stack overflow](https://stackoverflow.com/questions/37284423/glog-flag-redefined-error).  

I haven't dealt with vendoring before. 

Do you know where I'd add the appropriate flags to glide to ensure that dependencies are flattened?